tools/buildtools/packmol/packmol.xml

```diff
@@ -0,0 +1,196 @@
+<tool id="packmol" name="PACKMOL" version="1.18.013">
+    <description>Initial configurations for Molecular Dynamics Simulations by packing optimization</description>
+    <macros>
+        <import>macros_packmol.xml</import>
+    </macros>
+    <requirements>
+      <requirement type="package" version="1!18.013">packmol</requirement>
```

Comment:
wee, what is this, do they really have a ! in the version?

Reply:
yup that is someone else's package!